src/components/templates/Page.jsx

```diff
@@ -235,18 +232,15 @@ export const Page = ({
                   show={modalData.show}
                   square={links.square}
                 />
-                <LinksBox
-                  position={ssrDone && isMobile ? 'bottom-center' : links.position}
-                  zIndex={4}
-                  isPreviewMobile={isPreviewMobile}
-                >
+                <LinksBox position={links.position} zIndex={4} isPreviewMobile={isPreviewMobile}>
```

Comment:
Is this change just effective in Preview or is it on every public page? Questioning because public pages in mobile devices should render links only in the bottom-center.

Reply:
<Page> components is common both for preview and public pages.
Please make required changes because I might have overridden this during resolving conflicts between your commits and Joel's on my machine.

Reply:
Sure. I changed this component too and what you overwrote was my code :)